lib/test_formulae.rb

```diff
@@ -13,6 +13,28 @@ def initialize(tap:, git:, dry_run:, fail_fast:, verbose:)
 
       protected
 
+      def install_formula_from_bottle(formula_name, testing_formulae_dependents:, dry_run:)
+        bottle_filename = Dir.glob("#{formula_name}--*.#{Utils::Bottles.tag}.bottle*.tar.gz").first
+        if bottle_filename.blank?
+          if testing_formulae_dependents && !dry_run
+            raise "Failed to find bottle for '#{formula_name}'."
+          elsif !dry_run
+            return false
+          end
+
+          bottle_filename = "$BOTTLE_FILENAME"
```

Comment:
I don't understand what this is for.

Reply:
Lifted from existing code:

https://github.com/Homebrew/homebrew-test-bot/blob/430b676483f1b10752041c144e4c6d4d64d78ab0/lib/tests/formulae_dependents.rb#L36

Reply:
This seems like it would error always but happy to deal with this in a separate PR.